cpp/include/raft/sparse/linalg/detail/degree.cuh

```diff
@@ -90,9 +90,9 @@ RAFT_KERNEL coo_degree_scalar_kernel(
  * @param results: output row counts
  * @param stream: cuda stream to use
  */
-template <int TPB_X = 64, typename T>
+template <uint64_t TPB_X = 64, typename T, typename outT>
```

Comment:
This is threads_per_block, which cann't be larger than int. Was this intentional?